.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,122 @@
+name: Tag Repository for Release to Staging
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag (YYYY.MM.DD) to push. Defaults to current day"
+        required: false
```

Comment:
I think this should be required - tagging for the current day is an atypical use case - the typical case is today+2, but I think even it's clearer to require it

Reply:
The tag date is the date it will go to production so we can investigate historically when features actually went live.  Naming it as tagging date doesn't help with that, whereas you can find out in git history when a tag was created.

Reply:
Changing out tagging convention seems out of scope for this PR.  So, currently, we pretty much never create a tag named for the current day, so it's a footgun to default to it.